gilbreth_perception/package.xml

```diff
@@ -40,6 +40,18 @@
   <!-- Use test_depend for packages you need only for testing: -->
   <!--   <test_depend>gtest</test_depend> -->
   <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>pcl_conversions</build_depend>
+  <build_depend>pcl_ros</build_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>sensor_msgs</build_depend>
+  <build_depend>libpcl-all-dev</build_depend>
```

Comment:
Do you mean only this one is not needed:  <build_depend>libpcl-all-dev</build_depend> ?
Or those 4 build_depend are all not needed?